test/jit/test_upgraders.py

```diff
@@ -131,6 +135,27 @@ def test_func():
         version = self._load_model_version(loaded_func)
         self.assertTrue(version == 4)
 
+    @unittest.skipIf(not _is_upgraders_enabled(), "Skipping because upgraders are not enabled")
```

Comment:
When is `_is_upgraders_enabled()` true and when is it false?

Reply:
This was a guard to safely roll out our new upgraders design. It is controlled by a preprocessing boolean macro where the false enables your old design and true enables our new design. Right now, it is set to True